addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -551,9 +551,9 @@ export const editorCommands = {
         // Get the <font> nodes to color
         const selectedNodes = getSelectedNodes(editor.editable);
         const fonts = selectedNodes.flatMap(node => {
-            let font = closestElement(node, 'font');
+            let font = closestElement(node, 'font') ? closestElement(node, 'font') : closestElement(node, 'span');
```

Comment:
IIRC `closestElement` takes a selector as argument so giving it `"font, span[style]"` as argument should suffice and simplify this line. Tell me if that doesn't work.

Reply:
@dmo-odoo Yes, it's working.

Reply:
Hello @dmo-odoo , I have come across one problem with the last fix in this PR.
if we select like `<p> [ a <span font-size: 8px> b </span> c ]</p>` and apply color at that time result that I am getting is `<p> [ <font style="color: rgb(255, 0, 0)"> a <span font-size: 8px> <font style="color: rgb(255, 0, 0)"> b </font></span> c </font>]</p>`.
now if we want to remove color for only text that is contained in the span we will not able to remove it.

`let font = closestElement(node, 'font') ? closestElement(node, 'font') : closestElement(node, 'span');`
I have checked the same with the above code and I am not getting issues because here I am getting a result `<p> [ <font style="color: rgb(255, 0, 0)"> a <span font-size: 8px>  b </span> c </font>]</p>`

Reply:
I see, thank you. Does it work if you use this instead:
```js
let font = closestElement(node, 'font') || closestElement(node, 'span');
```
If it works then let's do that.

Reply:
@dmo-odoo Thanks for the suggestion. it's working properly.